packages/stylelint-copyright/src/index.ts

```diff
@@ -38,12 +47,12 @@ const plugin = stylelint.createPlugin(
         root.first.source?.start?.column === 1
       ) {
         const {text} = root.first;
-        if (text === secondaryOption.header) {
+        if (text === secondaryOption?.header) {
```

Comment:
Could you elaborate on this change? It should be working fine before.

Reply:
I noticed the header in above css files are not the same as the standard template, and I happened to run the `yarn lint` without the `.stylelintignore` file.

You are right, and I'd think unnecssary to change this file based on the ignored rulers.

Reply:
Let's revert this change, then, and focus the PR to fix the CSS import order

Reply:
done, see at d92db38286273c10e7eaf51d64ceb743b68e2df5